components/survey/Overview/prepareQuestions.js

```diff
@@ -0,0 +1,15 @@
+export const prepareQuestions = surveyResponse => {
```

Comment:
```suggestion
export const preparedQuestions = surveyResponse => {
```
As the purpose of this function seems to be `returning the prepared questions`, should rename it to `preparedQuestions` (or maybe `getPreparedQuestions`)? :)

Reply:
Thanks, renamed it to `getQuestionsFromSurvey` in 3739c999daf3ed824f99b52541f6c6dfc7f08236!